api-reference/10 UI Widgets/dxGantt/1 Configuration/resourceAssignments/resourceAssignments.md

```diff
@@ -12,4 +12,183 @@ Configures resource assignments.
+
+#####See Also#####
+- [Bind to File Systems](concept/05%20Widgets/File%20Manager/10%20Bind%20to%20File%20Systems/Bind%20to%20File%20Systems.md 'Documentation/Guide/Widgets/File_Manager/Bind_to_File_Systems/')
+
+---
+
+##### jQuery
+
+    <!-- tab: index.js -->
+    $(function() {
+        $("#gantt").dxGantt({
+            resourceAssignments: {
+                dataSource: resourceAssignments,
+                keyExpr: "key",
+                resourceIdExpr: "resourceKey",
+                taskIdExpr: "taskKey"
+            },
+            //...
+        });
+    });
+
+    <!-- tab: data.js -->
+    var resourceAssignments = [{
+        'key': 0,
+        'taskKey': 3,
+        'resourceKey': 1
+    },
+    // ...
+    ];    
+
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <dx-gantt ... >
+        <dxo-resource-assignments 
+            [dataSource]="resourceAssignments" 
+            keyExpr="key"
+            resourceIdExpr="resourceKey" 
+            taskIdExpr="taskKey">
+        </dxo-resource-assignments>
+        <!-- ... -->
+    </dx-gantt>
+
+    <!-- tab: app.component.ts -->
+    import { Component } from '@angular/core';
+    
+    @Component({
+        selector: 'app-root',
+        templateUrl: './app.component.html',
+        styleUrls: ['./app.component.css']
+    })    
+    
+    export class AppComponent {
+        resourceAssignments: ResourceAssignment[];
+        // ...
+
+        constructor(service: Service) {
+            this.resourceAssignments = service.getResourceAssignments();
+            // ...
+        }
+    }    
+    
+    <!-- tab: app.module.ts -->
+    import { BrowserModule } from '@angular/platform-browser';
+    import { NgModule } from '@angular/core';
+    import { AppComponent } from './app.component';
+    import { DxGanttModule } from 'devextreme-angular';
+    import { Service, ResourceAssignment, ... } from './app.service';
+
+    <!-- tab: app.service.ts -->
+    let resourceAssignments: ResourceAssignment[] = [{
```

Comment:
If you use a custom type `ResourceAssignment`, you should show what it looks like. For example, in the Gantt demo this is the following class: 

```
export class ResourceAssignment {
    id: number;
    taskId: number;
    resourceId: number;
}
```